dubbo-remoting/dubbo-remoting-zookeeper/src/main/java/org/apache/dubbo/remoting/zookeeper/zkclient/ZkClientWrapper.java

```diff
@@ -43,28 +41,26 @@
     private long timeout;
     private ZkClient client;
     private volatile KeeperState state;
-    private ListenableFutureTask<ZkClient> listenableFutureTask;
+    private CompletableFuture<ZkClient> completableFuture;
     private volatile boolean started = false;
+    private static final ExecutorService executor = new ThreadPoolExecutor(0, 10, 60L, TimeUnit.SECONDS, new SynchronousQueue<Runnable>(), new NamedThreadFactory("DubboMonitorCreator", true));
 
 
     public ZkClientWrapper(final String serverAddr, long timeout) {
         this.timeout = timeout;
-        listenableFutureTask = ListenableFutureTask.create(new Callable<ZkClient>() {
-            @Override
-            public ZkClient call() throws Exception {
+        completableFuture = CompletableFuture.supplyAsync(()->{
                 return new ZkClient(serverAddr, Integer.MAX_VALUE);
-            }
-        });
+        },executor);
     }
 
     public void start() {
         if (!started) {
-            Thread connectThread = new Thread(listenableFutureTask);
+            Thread connectThread = new Thread();
```

Comment:
I think `Thread connectThread = new Thread();` is no longer needed here, because we already started `ZkClient` in an async thread by invoking `CompletableFuture.supplyAsync`